tests/README.md

```diff
@@ -54,12 +54,20 @@ tests:
    tasty's pattern feature. For example:
 
    ```bash
-   cabal test --test-options="--pattern=Testsuite/Singletons/*"
+   cabal v2-test --test-options="--pattern=Testsuite/Singletons/*"
    ```
 
    runs all tests in the Testsuite/Singletons branch of the test tree.
    SingletonsTestSuite module defines structure of the test tree.
 
+ * If you modify `singletons`, you may cause the actual output of some tests to
+   change. If these changes are what you intended, you can accept the new
+   output by running the following command:
+
+   ```bash
+   cabal v2-test --test-options="--accept"
```

Comment:
the v2 prefix is redundant with the recently released cabal 3.0

Reply:
My original thought was that this command would be backwards-compatible with older `cabal-install`s, but I now realize that GHC 8.8 requires `cabal-install-3.0` or later anyway.

I've shortened this to just `cabal test`.